### PR TITLE
2951_API_end_point_to_support_polling

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

File: server/providers/pouchdb.js [hunk 1]
```diff
@@ -298,6 +298,48 @@ const pouchDb = (app, io, pouchOptions) => {
     ).catch(() => res.send([]))
   })
 
+  app.get('/:wargame/lastDoc/:id?', async (req, res) => {
+    const databaseName = checkSqliteExists(req.params.wargame)
+
```
Comment (IanMayo) [inline server/providers/pouchdb.js:303]: @lilitkarapetyan - in the above call to `checkSqliteExists`, shouldn't we use the `:wargame` from the end-point?

**Comment by IanMayo** [comment]
@lilitkarapetyan - we'll need this for CouchDb too, right?

**Comment by lilitkarapetyan** [comment]
> @lilitkarapetyan - we'll need this for CouchDb too, right?

Yes, I am working on it now.

**Review by IanMayo — APPROVED** [review]
Tested and working

**Comment by lilitkarapetyan** [comment]
fixes #2951

**Pull request merged**